index.adoc

```diff
@@ -98,6 +98,6 @@ include::./contributing/review_reviewer.adoc[leveloffset=+3]
 
 // 
 
-include::./general_docs/glossary.adoc[leveloffset=+1]
+//include::./general_docs/glossary.adoc[leveloffset=+1]
 
-include::./general_docs/bibliography.adoc[leveloffset=+1]
+//include::./general_docs/bibliography.adoc[leveloffset=+1]
```

Comment:
What has this to do with this PR? 

These files are still included in the repository. Are they tbd or not needed any longer?

Reply:
The bibliography is part of the standard set of general docs .  Same is true for the glossary. See also #27.
 
However, there is content for a bibliography or a glossary topic right now. Thus I commented it out.
I created an issue to fill the topic in a future release (#36, #37).

Reply:
You mean there is **not**, right? 
The files are filled with lorem ipsum at this point. Shouldn't they be removed if there is no content?

Reply:
You are right of course, there is **no** content right now.
I figured that we could leave the topics as they are, since they are not included in the final product.

The general idea behind commenting out topics, rather than deleting them, was that we still keep the intended structure.
It may also help avoid merge conflicts, Commenting out a line is a less drastic change than deleting the line and the file.

Reply:
Yes, does work, but I think its not very elegant. Especially filled with lotem ipsum. Does not looks intentional and like something we forgotten. 

You could also have it included in the mapping file and have it empty with just a comment stating that there is no content yet.

Reply:
Anyways. This is very low prio..